src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -515,6 +516,16 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
                 options = options.WithConcurrentBuild(false);
             }
 
+#if Test_IOperation_Interface
```

Comment:
Please keep preprocessor definitions all upper case.  This is historically howt tehy are represented.